include/BaslerCamera.h

```diff
@@ -116,8 +116,8 @@ class LIBBASLER_API Camera
     void setLatTime(double  lat_time);
     void getLatTime(double& lat_time);
 
-    void getExposureTimeRange(double& min_expo, double& max_expo) const;
-    void getLatTimeRange(double& min_lat, double& max_lat) const;    
+    void getExposureTimeRange(double& min_expo, double& max_expo);
+    void getLatTimeRange(double& min_lat, double& max_lat);
```

Comment:
I am not able to put back the `const` qualifier because these methods now call `_getBasler()`. This method cannot be `const` because it creates an AutoMutex which needs a non const mutex from the condition.
Any ideas?